app/helpers/feedback_helper.rb

```diff
@@ -1,8 +1,8 @@
 module FeedbackHelper
   def visit_purpose_options
-    [['Find a job in teaching', 1],
-     ['List a teaching job on the service', 2],
-     ['Something else (tell us in the box below)', 3]]
+    [['Find a job in teaching', :find_teaching_job],
```

Comment:
These should probably be translations I reckon

Reply:
:+1: